Printers/Canon/template_canon_oce_plotwave/6.0/template_canon_oce_plotwave.yaml

```diff
@@ -0,0 +1,547 @@
+zabbix_export:
+  version: '6.4'
```

Comment:
Please change the directory according to the template version.

Reply:
I changed

Reply:
hh